README.md
# 3kh0.github.io

![3kh0.github.io](https://socialify.git.ci/3kh0/3kh0.github.io/image?description=1&descriptionEditable=This%20is%203kh0.github.io%20v3%20the%20better%20unblocked%20games%20website!&font=Source%20Code%20Pro&forks=1&issues=1&language=1&logo=https%3A%2F%2F3kh0.github.io%2Ffavicon.ico&name=1&owner=1&pattern=Circuit%20Board&pulls=1&stargazers=1&theme=Dark)

Welcome to 3kh0.github.io, your best option for unblocked games on the internet. This is the GitHub repo which is used to host the website. If you want to use the games on here on your website, please message me on discord and we can talk! `3kh0_#1791`

> **Note**
> **If you do fork this repo, please give the orginal repo a star! It helps out alot and only 30% of people who fork give me a star.**

[![Vulnerability](https://github.com/3kh0/3kh0.github.io/actions/workflows/codeql-analysis.yml/badge.svg?branch=main)](https://github.com/3kh0/3kh0.github.io/actions/workflows/codeql-analysis.yml) [![pages-build-deployment](https://github.com/3kh0/3kh0.github.io/actions/workflows/pages/pages-build-deployment/badge.svg?branch=main)](https://github.com/3kh0/3kh0.github.io/actions/workflows/pages/pages-build-deployment)

## Features

- Site-wide tab cloaking
- Easy to deploy
- Clean UI
- Cross platform
- Easy to use
- Light on host resources
- And so much more!


## Live deployment

You can access this project using the link below

[https://3kh0.github.io](https://3kh0.github.io)


## Host the website

Below are some of the options you can use to deploy your own copy of the site. **If you do not know what you are doing, it is suggested that you use a link someone is currently hosting!**

[![Run on Replit](https://binbashbanana.github.io/deploy-buttons/buttons/remade/replit.svg)](https://github.com/3kh0/3kh0.github.io-replit)
[![Remix on Glitch](https://binbashbanana.github.io/deploy-buttons/buttons/remade/glitch.svg)](https://glitch.com/edit/#!/import/github/3kh0/3kh0.github.io)
[![Deploy with Vercel](https://binbashbanana.github.io/deploy-buttons/buttons/remade/vercel.svg)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2F3kh0%2F3kh0.github.io) 
[![Deploy with Netlify](https://binbashbanana.github.io/deploy-buttons/buttons/remade/netlify.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/3kh0/3kh0.github.io)
[![Deploy to Render](https://binbashbanana.github.io/deploy-buttons/buttons/remade/render.svg)](https://render.com/deploy?repo=https://github.com/3kh0/3kh0.github.io)
[![Deploy to Cyclic](https://binbashbanana.github.io/deploy-buttons/buttons/remade/cyclic.svg)](https://app.cyclic.sh/api/app/deploy/3kh0/3kh0.github.io)

> **Note**
> If you have any problems please open a issue!

## Contributers

Thank you to all of these amazing people for helping work on this project!

![Contrib](https://contrib.rocks/image?repo=3kh0/3kh0.github.io#)

And the people who worked on v2!

![Contrib](https://contrib.rocks/image?repo=3kh0/website-v2#)

And the people who worked on v1!

![Contrib](https://contrib.rocks/image?repo=3kh0/website-v1#)

## Contributing

Contributions are always welcome!

See [`contributing.md`](https://github.com/3kh0/3kh0.github.io/blob/main/.github/CONTRIBUTING.md) for ways to get started.

Please adhere to this project's [`code of conduct`](https://github.com/3kh0/3kh0.github.io/blob/main/.github/CODE_OF_CONDUCT.md).


## Badges

![GitHub issues](https://img.shields.io/github/issues/3kh0/3kh0.github.io?logo=github&style=flat-square) 
![GitHub pull requests](https://img.shields.io/github/issues-pr/3kh0/3kh0.github.io?label=Pull%20requests&logo=github&style=flat-square) 
![GitHub](https://img.shields.io/github/license/3kh0/3kh0.github.io?label=Licence&logo=github&style=flat-square) 
![GitHub forks](https://img.shields.io/github/forks/3kh0/3kh0.github.io?label=Forks&logo=github&style=flat-square) 
![GitHub Repo stars](https://img.shields.io/github/stars/3kh0/3kh0.github.io?color=yellow&label=Stars&logo=github&style=flat-square) 
![GitHub repo size](https://img.shields.io/github/repo-size/3kh0/3kh0.github.io?label=Repo%20size&logo=github&style=flat-square) 

## Run Locally

First clone the project using command line

```bash
git clone https://github.com/3kh0/3kh0.github.io.git
```

or if you have GitHub CLI

```bash
gh repo clone 3kh0/3kh0.github.io
```

Since this project does not require a backend, for the best experience I suggest using [XAMPP](https://www.apachefriends.org/), cloning the repo into the Apache Server directory at `C:\xampp\htdocs` (for windows), setting the port to something like 8080, opening up a browser and going to [`localhost:8080`](http://localhost:8080)

> **Warning**
> Please make your you know what you are doing, you could delete your entire drive if you are not careful!

## License


© Echo, 2022. Unauthorized use and/or duplication of this material without permission from Echo is strictly prohibited. Excerpts and links may be used, provided that clear credit is given with a link to the original content.

This code is licensed under the **Apache-2.0 License**. I ask you that you do not copy the code in the repo what-so-ever; I take time to build this and I do not need someone just yoinking all of my games for ones site. If you have any questions please open a issue!

> **Note**
> You can view the full license [here](https://github.com/3kh0/3kh0.github.io/blob/main/LICENSE)!


## Previous versions

Here are some of the previous versions of this project. To give you a
sense, this is v3

- [Website v1](https://github.com/3kh0/website-v1)
- [Website v2](https://github.com/3kh0/website-v2)

## More badges

[![forthebadge](https://forthebadge.com/images/badges/compatibility-club-penguin.svg)](https://forthebadge.com)
[![forthebadge](https://forthebadge.com/images/badges/contains-tasty-spaghetti-code.svg)](https://forthebadge.com)
[![forthebadge](https://forthebadge.com/images/badges/it-works-why.svg)](https://forthebadge.com)
[![forthebadge](https://forthebadge.com/images/badges/made-with-crayons.svg)](https://forthebadge.com)
[![forthebadge](https://forthebadge.com/images/badges/mom-made-pizza-rolls.svg)](https://forthebadge.com)
[![forthebadge](https://forthebadge.com/images/badges/not-a-bug-a-feature.svg)](https://forthebadge.com)
[![forthebadge](https://forthebadge.com/images/badges/powered-by-black-magic.svg)](https://forthebadge.com)
[![forthebadge](https://forthebadge.com/images/badges/powered-by-pull-requests.svg)](https://forthebadge.com)
[![forthebadge](https://forthebadge.com/images/badges/works-on-my-machine.svg)](https://forthebadge.com)
[![forthebadge](https://forthebadge.com/images/badges/uses-brains.svg)](https://forthebadge.com)
[![forthebadge](https://forthebadge.com/images/badges/0-percent-optimized.svg)](https://forthebadge.com)


dump/changelog.txt
░█████╗░██╗░░██╗░█████╗░███╗░░██╗░██████╗░███████╗  ██╗░░░░░░█████╗░░██████╗░
██╔══██╗██║░░██║██╔══██╗████╗░██║██╔════╝░██╔════╝  ██║░░░░░██╔══██╗██╔════╝░
██║░░╚═╝███████║███████║██╔██╗██║██║░░██╗░█████╗░░  ██║░░░░░██║░░██║██║░░██╗░
██║░░██╗██╔══██║██╔══██║██║╚████║██║░░╚██╗██╔══╝░░  ██║░░░░░██║░░██║██║░░╚██╗
╚█████╔╝██║░░██║██║░░██║██║░╚███║╚██████╔╝███████╗  ███████╗╚█████╔╝╚██████╔╝
░╚════╝░╚═╝░░╚═╝╚═╝░░╚═╝╚═╝░░╚══╝░╚═════╝░╚══════╝  ╚══════╝░╚════╝░░╚═════╝░

(btw if you can not read what is says above, it is just saying changelog!)

Here is where I will post updates on when a new game is added or something changed. Besure to check back often for any sneakpeaks or other goodies! If I can I will try to setup a RSS feed for all of those who use that. Coming soon!

Besure to also join the discord: https://discord.gg/wv6huJAwEv

NOTE: Make sure you refresh the page! Sometimes you will not get the lastest update!

+ Means something was added
^ Means something was upgraded or changed
- Means something was removed
------------------------------------------------------------
Oct, 30
+ Added OvO
+ Added Fancy Pants Adventures
+ Added Kitchen gun game
+ Added Tiny Islands
------------------------------------------------------------
Oct, 29
+ Added Rolling Forests
+ Added Waterworks!
------------------------------------------------------------
Oct, 28
+ Added Flippy Fish
+ Added Frying Nemo
^ Added more reviews
------------------------------------------------------------
Oct, 27
+ Added Offline Paradise
+ Added Replit Deployment!
------------------------------------------------------------
Oct, 24
+ Added more reviews
------------------------------------------------------------
Oct, 23
+ Added Fire Boy and Water Girl
+ Added Super Mario Construct
+ Added Yoshi's Fabrication Station
------------------------------------------------------------
Oct, 20
+ Added Doctor Acorn 2
+ Added Twitch Tetris
+ Added quick access to site settings
^ Improved the 404 page with more fun stats
- Removed the ip request on 404 page
------------------------------------------------------------
Oct, 19
^ Added more reviews
------------------------------------------------------------
Oct, 17
+ Added Sandboxels
------------------------------------------------------------
Oct, 14
^ Fixed a bug
------------------------------------------------------------
Oct, 13
+ Added Space Garden
+ Added Rabbit Samurai
+ Added Rabbit Samurai 2
^ Added more splash texts
^ Added more reviews
------------------------------------------------------------
Oct, 12
+ Added Dante
+ Added SHUTTLEDECK
+ Added Edge Lets surf
+ Added Ninja vs Evil Corp
+ Added Sleeping beauty
+ Added Black hole square
+ Added A Dark Room
^ Added more splash texts
------------------------------------------------------------
Oct, 11
+ Added xx142-b2exe
------------------------------------------------------------
Oct, 10
^ Added more splash texts
------------------------------------------------------------
Oct, 9
+ Added Dragons vs Bricks
+ Added Gravity Soccer
+ Added Go Ball
+ Added Slope Ball
+ Added X Trial Racing
+ Added even more reviews (please keep it coming!)
------------------------------------------------------------
Oct, 7
- Removed fish client
------------------------------------------------------------
Oct, 5
^ Added more splash texts
------------------------------------------------------------
Oct, 4
+ Added Stickman boost
^ Added more splash texts
------------------------------------------------------------
Oct, 2
^ Added more splash texts
------------------------------------------------------------
Oct, 1
+ Added 2048 Multitask
+ Added Fish Hack Client (Eaglercraft)
+ Added search bar to projects list
^ Changed the reviews so they scrolled
- Remove #VoteForJude splash
------------------------------------------------------------
Sep, 15
+ Added ext remover
------------------------------------------------------------
Sep, 13
+ Added more splash texts
^ Optimized all images
------------------------------------------------------------
Sep, 10
+ Added Glass City
+ Added Station 141
+ Added VELOCE
+ Added Core Ball
+ Added Protektor
+ Added Drive Mad (kinda)
^ Fixed error on about:blank embedder looking weird
^ Added a review from food to homepage
^ Fix issue #40
------------------------------------------------------------
Sep, 9
+ Added Tactical Weapon Pack 2
------------------------------------------------------------
Sep, 8
+ Added GOODNIGHT
------------------------------------------------------------
Sep, 7
+ Added 100ng
+ Started the open beta for our discord in browser plugin!
------------------------------------------------------------
Sep, 5
+ Added Random Reddit meme (going to make seperate repo for this soon)
+ Added Om Nom Bounce
------------------------------------------------------------
Aug, 29
+ Added Krunker
+ Added Precision Client
------------------------------------------------------------
Aug, 28
+ Updated site to v3
------------------------------------------------------------
May, 23
+ Added Subway Surfers
------------------------------------------------------------
May, 14
+ Added 50+ Games (Thank you ThatOnePerson!)
------------------------------------------------------------
May, 13
+ Added gimme the airpod
------------------------------------------------------------
May, 12
+ Added 1v1, made by Elijah
+ Added Big Tower Tiny Square
^ Set tab spacing to 2 on index
------------------------------------------------------------
May, 10
+ Added CircloO
------------------------------------------------------------
May, 6
+ Added Wolfenstein 3D
+ Added Hextris
^ Fixed issue with Eaglercraft not working
------------------------------------------------------------
May, 5
New discord link: https://discord.gg/wv6huJAwEv
------------------------------------------------------------
May, 4

js/main.js
// Hello there!
//
// If you want to add my games to your site, please reach out at my email: echo-the-coder@tuta.io, or discord: 3kh0_#1791

console.warn(
  "%cNote!",
  "color: purple; font-weight: 600; background: yellow; padding: 0 5px; border-radius: 5px",
  "If you want to add my games to your site, please reach out at my email: echo-the-coder@tuta.io\nPlease do not just add them without asking me first! Thank you!"
);

function script(text) {
  console.log("%cScript Injection", "color: cyan; font-weight: 600; background: black; padding: 0 5px; border-radius: 5px", text);
}

// ====================================
// SCRIPT INJECTION
// ====================================

var gaenabled = window.localStorage.getItem("ga");
if (gaenabled == "false") {
  script("Skipped GA injection because it is disabled by the user.");
} else {
  const gascript = document.createElement("script");
  gascript.setAttribute("async", "");
  gascript.setAttribute("src", "https://www.googletagmanager.com/gtag/js?id=G-98DP5VKS42");
  const inlinegascript = document.createElement("script");
  inlinegascript.innerHTML = `window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());
      gtag('config', 'G-98DP5VKS42');`;
  document.head.append(gascript, inlinegascript);
  script("Injected script 1/4");
}

const tabCloak = document.createElement("script");
tabCloak.setAttribute("src", "/js/tab_cloak.js");
document.head.append(tabCloak);
script("Injected script 2/4");

const arc = document.createElement("script");
arc.setAttribute("async", "");
arc.setAttribute("src", "https://arc.io/widget.min.js#eRPHFgiC");
document.head.append(arc);
script("Injected script 3/4");

const adblock = document.createElement("script");
adblock.setAttribute("src", "https://fundingchoicesmessages.google.com/i/pub-5756835229788588?ers=1");
adblock.setAttribute("nonce", "yibq-w_TR5NOCRWsU-VL0Q");
adblock.setAttribute("async", "");
document.head.append(adblock);
script("Injected script 4/4");


js/search.js
function startSearch() {
  var input, filter, ul, li, a, i, b, dropDown;
  input = document.getElementById("mySearch");
  dropDown = document.getElementById("select");
  filter = input.value.toUpperCase();
  ul = document.getElementById("search");
  li = ul.getElementsByTagName("a");
  for (i = 0; i < li.length; i++) {
    a = li[i].getElementsByTagName("h1")[0];
    if (a.innerHTML.toUpperCase().indexOf(filter) > -1) {
      li[i].style.display = "";
    } else {
      li[i].style.display = "none";
    }
    
    if (dropDown.value == "n/a") {
      continue; //if no selection, disregard everything below this line in the for loop
    }
    
    b = a.getElementsByClassName("badge");
    if ((b ? b[0].innerHTML.includes("New") : 0) && (dropDown.value == "new")) {
      li[i].style.display = "";
    } else if ((b ? b[0].innerHTML.includes("Hot") : 0) && (dropDown.value == "hot")) {
      li[i].style.display = "";
    } else if (b.length == 0) && (dropDown.value == "n/a")) {
      li[i].style.display = "";
    } else {
      console.log("this will probably break lol");
    }
  }
}


js/subtitle.js
// script to generate random greetings

function changeSub(num) {
  document.getElementById("subtitle").innerText = say[num];
}


var say = [];
say[0] = "Welcome.";
say[1] = "Hello!";
say[2] = "Does this even work?";
say[3] = "Welcome to my rad website";
say[4] = "Made with HTML!";
say[5] = "Now with buggy games!";
say[6] = "You get a cookie!";
say[7] = "Remember your manners!";
say[8] = "Dogs are cute";
say[9] = "Ok, and I know your address now!";
say[10] = "Beep boop! Are you a robot?";
say[11] = "Did you know I have a discord server?";
say[12] = "How many sides does a circle have?";
say[13] = "1 + 1 = 3";
say[14] = "You can read btw";
say[15] = "Made you look!";
say[16] = "Never gonna give you up, Never gonna let you down...";
say[17] = "Redbull gives you wings...";
say[18] = "What is your highscore on Swerve?";
say[19] = "Since 2018!";
say[20] = "Now with 2X more bugs!";
say[21] = "12345 is not a good password";
say[22] = "This took to long to make";
say[23] = "69 hehe funny number";
say[24] = "Ahhhhhhhhh";
say[25] = "Awesome!";
say[26] = "As seen on TV!";
say[27] = "doot doot";
say[28] = "100% Fresh!";
say[29] = "LOL";
say[30] = "Cold as ice!";
say[31] = "Why are you looking at me?";
say[32] = "Made in the US!";
say[33] = "Made by 3kh0!";
say[34] = "Reload the page NOW!";
say[35] = "Downloading your passwords...";
say[36] = "Running 3kh0.exe";
say[37] = "WARNING: You may lose braincells if you proceed!";
say[38] = "Please go away. Thx";
say[39] = "Made with GitHub!";
say[40] = "Made with VS code!";
say[41] = "This is a splash text!";
say[42] = "100% Orgainic!";
say[43] = "Now with Pac-Man!";
say[44] = "Now with Polybranch!";
say[45] = "Now with JavaScript!";
say[46] = "Do you bite your ice cream?";
say[47] = "Reddit: u/3kh0_reddit";
say[48] = "Discord: https://discord.gg/wv6huJAwEv";
say[49] = "This was not ment to be seen. :o";
say[50] = "Does anyone know how to stand up?";
say[51] = "69420";
say[52] = "Me too bro.";
say[53] = "#Relatable";
say[54] = "2 + 2 is 4 - 1 is 3 Quick mafs";
say[55] = "Feels bad man.";
say[56] = "Now with games that are unblocked?";
say[57] = "Now with great games!";
say[58] = "How do you play tic-tac-toe?";
say[59] = "Fortnite is cringe";
say[60] = "You got games on your phone?";
say[61] = "I am not gonna say anything.";
say[62] = "Now with Minecraft!";
say[63] = "Welcome to v3.0!";
say[64] = "Your momma jokes are cringe";
say[65] = "Now with a new splash!";
say[66] = "You just dont get it do you?";
say[67] = "Powered by hamsters!";
say[68] = "*insert cool explosion*";
say[69] = "Efficiency is key";
say[70] = "New update???";
say[71] = "We have answers!";
say[72] = "aaaaaaaaaaaaaaaaaaa";
say[73] = "Yooooooooooooooooooooo";
say[74] = "Your ip is 69.420.69.420";
say[75] = "he on x-games mode";
say[76] = "Got to go fast!";
say[77] = "GAS GAS GAS GOTTA STEP ON THE GAS!";
say[78] = "3kh0.github.io would like to view your browser history";
say[79] = "For legal reasons, that's a joke!";
say[80] = "It has been 84 years...";
say[81] = "Made by Gucci, selling for $69420";
say[82] = "There is someone looking over your shoulder";
say[83] = "Electric!";
say[84] = "How many different ones are there?!?!";
say[85] = "NERDDDDDDDDDDDDDDDDDD";
say[86] = "My iPad passcode is 563543 btw";
say[87] = "Get out of my room, I'm not going to let you in!";
say[88] = "I'm not a robot!";
say[89] = "I'm not a human!";
say[90] = "Sponsered by RAID SHADOW LEGENDS!";
say[91] = "Sponsored by Honey!";
say[92] = "Sponsored by 3kh0!";
say[93] = "Sponsored by your mom!";
say[94] = "Sponsored by your dad!";
say[95] = "Sponsored by your mom's dad!";
say[96] = "Sponsored by your dad's mom!";
say[97] = "Sponsored by your mom's dad's mom!";
say[98] = "Sponsored by your dad's mom's dad!";
say[99] = "Sponsored by your mom's dad's mom's dad!";
say[100] = "Sponsored by your dad's mom's dad's mom!";
say[101] = "Sponsored by your mom's dad's mom's dad's mom!";
say[102] = "Sponsored by your dad's mom's dad's mom's dad!";
say[103] = "Sponsored by your mom's dad's mom's dad's mom's dad!";
say[104] = "Sponsored by your dad's mom's dad's mom's dad's mom!";
say[105] = "Sponsored by your mom's dad's mom's dad's mom's dad's mom!";
say[106] = "Sponsored by your dad's mom's dad's mom's dad's mom's dad!";
say[107] = "Sponsored by your mom's dad's mom's dad's mom's dad's mom's dad!";
say[108] = "Sponsored by your dad's mom's dad's mom's dad's mom's dad's mom!";
say[109] = "Sponsored by your mom's dad's mom's dad's mom's dad's mom's dad's mom!";
say[110] = "Sponsored by your dad's mom's dad's mom's dad's mom's dad's mom's dad!";
say[111] = "Sponsored by your mom's dad's mom's dad's mom's dad's mom's dad's mom's dad!";
say[112] = "Sponsored by your dad's mom's dad's mom's dad's mom's dad's mom's dad's mom!";
say[113] = "Sponsored by your mom's dad's mom's dad's mom's dad's mom's dad's mom's dad's mom!";
say[114] = "Sponsored by your dad's mom's dad's mom's dad's mom's dad's mom's dad's mom's dad!";
say[116] = "Sponsored by your dad's mom's dad's mom's dad's mom's dad's mom's dad's mom's dad's mom!";
say[117] = "Sponsored by your mom's dad's mom's dad's mom's dad's mom's dad's mom's dad's mom's dad!";
say[118] = "Sponsored by your dad's mom's dad's mom's dad's mom's dad's mom's dad's mom's dad's mom's dad!";
say[119] = "Sponsored by your mom's dad's mom's dad's mom's dad's mom's dad's mom's dad's mom's dad's mom!";
say[120] = "Sponsored by your dad's mom's dad's mom's dad's mom's dad's mom's dad's mom's dad's mom's dad's mom!";
say[121] = "Sponsored by your mom's dad's mom's dad's mom's dad's mom's dad's mom's dad's mom's dad's mom's dad!";
say[122] = "Sponsored by your dad's mom's dad's mom's dad's mom's dad's mom's dad's mom's dad's mom's dad's mom's dad!";
say[123] = "Sponsored by your mom's dad's mom's dad's mom's dad's mom's dad's mom's dad's mom's dad's mom's dad's mom!";
say[124] = "Piplup is cute";
say[125] = "3kh0 is a bozo";
say[126] = "Nothing has been said yet!";
say[127] = "I'm not a bird!";
say[128] = "I'm not a fish!";
say[129] = "I'm not a cat!";
say[130] = "I'm not a dog!";
say[131] = "I'm not a horse!";
say[132] = "I'm not a cow!";
say[133] = "I'm not a sheep!";
say[134] = "I'm not a goat!";
say[135] = "I'm not a chicken!";
say[136] = "I'm not a pig!";
say[137] = "I'm not a Piplup!";
say[138] = "I'm not a 3kh0!";
say[139] = "Powered by 128 Gigs of RAM!";
say[140] = "Powered by 1 Gig of RAM!";
say[141] = "I want to bite you!";
say[142] = "I want to eat you!";
say[143] = "I want to drink you!";
say[144] = "I want to devour you!";
say[145] = "I'm Slim Shady, yes, I'm the real Shady!";
say[146] = "I'm not a real Shady!";
say[147] = "I'll wear your granddad's clothes";
say[148] = "gooooooooooodbye";
say[149] = "You're a real jerk!";
say[150] = "w h a t";
say[151] = "what the dog doin?";
say[152] = "what are you doing?";
say[153] = "Math is kinda boring";
say[154] = "XD";
say[155] = "I'm gonna pop some tags";
say[156] = "Ka-ching!";
say[157] = "Kablam!";
say[158] = "Shoutout to 4chan!";
say[159] = "Shoutout to 4chan's mods! Wait, there are not any!";
say[160] = "thx stackoverflow!";
say[161] = "are you a gaemer?";
say[162] = "mom get out of my room i am playing minecraft!";
say[163] = "This is a test message!";
say[164] = "It is time to get funky!";
say[165] = "people like to barn fish?";
say[166] = "Why is the sky blue?";
say[167] = "Can you please burp?";
say[168] = "Some things are better left unspoken.";
say[169] = "What is your favorite color?";
say[170] = "What is your favorite animal?";
say[171] = "What is your favorite food?";
say[172] = "What is your favorite drink?";
say[173] = "What is your favorite band?";
say[174] = "What is your favorite song?";
say[175] = "What is your favorite movie?";
say[176] = "What is your favorite book?";
say[177] = "What is your favorite sport?";
say[178] = "What is your favorite game?";
say[179] = "What is your favorite TV show?";
say[180] = "What is your favorite TV character?";
say[181] = "SOMEONE PLEASE HELP ME!";
say[182] = "I'm a little teapot.";
say[183] = "I'm a little teapot, short and stout.";
say[184] = "We didn't start the fire! It was always burning, since the world's been turning!";
say[185] = "Don't worry, I'm not a firetruck.";
say[186] = "Don't ask questions if you can't handle the answers!";
say[187] = "I'm not a firetruck, I'm a firetruck.";
say[188] = "If you're not careful, you'll burn your house down.";
say[189] = "If you are using a computer, you are a computer.";
say[190] = "If you are using Microsoft Edge, you need some help.";
say[191] = "Living in space is like living in a hotel.";
say[192] = "Living in Spain without the S";
say[193] = "Living in Spain without the S is a bad idea.";
say[194] = "Shoutout to Piplup!";
say[195] = "Shoutout to 3kh0!";
say[196] = "Woo! /g/";
say[197] = "youtube.com/watch?v=dQw4w9WgXcQ";
say[198] = "youtube.com/c/shadrYT";
say[199] = '"Pretend inferiority and encourage his arrogance." －Sun Tzu';
say[200] = "Help im locked in a data center!";
say[201] = "5261796D6F6E6420486F6C74";
say[202] = "gif bezos";
say[203] = "God i just hate reality";
say[204] = "Please stop posting furry images.";
say[205] = "i wanted free vbucks :(";
say[206] = "i want to be a cat";
say[207] = "i want to be a dog";
say[208] = "i want to be a horse";
say[209] = "i want to be a cow";
say[210] = "i want to be a sheep";
say[211] = "i want to be a goat";
say[212] = "i want to be a chicken";
say[213] = "i want to be a pig";
say[214] = "i want to be a piplup";
say[215] = "i want to be a 3kh0";
say[216] = "i want to be a bozo";
say[217] = "i want to be a gaemer";
say[218] = "Is this the United Airlines costumer service center?";
say[219] = "we will steal your china and sell it to ip";
say[220] = "Never half-ass two things. Whole-ass one thing.";
say[221] = "bottem text";
say[222] = "top text";
say[223] = "Why do they call it oven if you oven the cold food of out hot eat the food";
say[224] = "furryegirl";
say[225] = "No fun allowed";
say[226] = "Your toes will be pulled tonight";
say[227] = "Someone was too lazy to give it a title";
say[228] = "i do love buying birth certificates from the black market! - Piplup"
say[229] = "'If you are trash go cry to Fortnite you loser' - Sun Tzu, Art of war"
say[230] = "undefined"
say[231] = "i wanted free vbucks :("
say[232] = "Avocados \ud83e\udd51 from Mexico \ud83c\uddf2\ud83c\uddfd"
say[233] = "i realised i haven't blinked in like half an hour and now blinking feels uncomfortable";
say[234] = "reload for another message";
say[235] = "Boost the discord server for a very cool role!";
say[236] = "Why does the discord server exist";
say[237] = "I purchased a baby clown from the Russian terrorist black market.";
say[238] = "He stomped on his fruit loops and thus became a cereal killer.";
say[239] = "Technoblade never dies!";
say[240] = "And just to think I knew him before he was famous!";
say[241] = "Trust me, I did not copy Minecraft with this!";
say[242] = "It is everyday bro!";
say[243] = "That proves you are unusual";
say[244] = "What's my age again?";
say[245] = "If you think you are dumb, it is true";
say[256] = "This is the 5th time this week, and it's only Monday!";
say[257] = "You bum";
say[258] = "Roblox? Cringe!";
say[259] = "Toss the Turtle is pretty fun";
say[260] = "Mission failed, we will get them next time!";
say[261] = "NUKE INCOMING!";
say[262] = "I spent too much time making these";
say[263] = "Too cool for school!";
say[264] = "Pringles";
say[265] = "Oh heck nah!";
say[266] = "Hoo hoo hoo!";
say[267] = "Light mode eww!";
say[268] = "pretzels";
say[269] = "Please game responsibly";
say[270] = "burbur";
say[271] = "Apple bottom jeans!";
say[272] = "We out here!";
say[273] = "what a gaemer";
say[274] = "Please enjoy responsibly";
say[275] = "Can you really buy land on the moon?";
say[276] = "Breaking news! Your computer will blow up in 3...";
say[277] = "Gaming since 2020!";
say[278] = "Hacker man!";
say[279] = "Your fortune is...";
say[280] = "Let's just hope you dont get caught doing this...";
say[281] = "Site shutting down in 2 days...jk";
say[282] = "This is not the first time you did this...";
say[283] = "baba booey";
say[284] = "Can't stop me now!";
say[285] = "beans";
say[286] = "OwO";
say[287] = "UwU";
say[288] = "._.";
say[289] = "null";
say[290] = "It's over 100!";
say[291] = "Get to work!"
say[292] = "Cats or Dogs?";
say[293] = "[error] var braincells is to low";
say[294] = "DM Echo if you want to see your name here!";
say[295] = "shadrYT on top!"
say[296] = "Sub to shadrYT!"
say[297] = "Free robux! Wait who even plays that game eww";
say[298] = "Did you know that there are other JAWS movies, yeah they are bad!";
say[299] = "This is the 300th splash text!";
say[300] = "Extra credit be like";
say[301] = "Blue's Potatos are superior";
say[302] = "Have you ever had a dream that you, um, you had, your, you- you could, youll do, you- you wants, you, you could do so, you- youll do, you could- you, you want, you want them to do you so much you could do anything?";
say[303] = "[object Object]";
say[304] = "Error 404, Games not found";
say[305] = "To the person who invented zero, thanks for nothing";
say[306] = "Piplup gets no b*****s haha (-piplup's sister)";
say[307] = "mogus";
say[308] = "Toto, I've a feeling we're not in Kansas anymore";
say[309] = "You keep using that word. I do not think it means what you think it means.";
say[310] = "c h a o s";
say[310] = "a- amogu- *gets shot with laser*";
say[311] = "potato";
say[312] = "Ping: 87654ms";
say[313] = "Ping: 1ms";
say[314] = "69 x 21 + 420 + 911 - 760 = 2020";
say[315] = "Aspaceman is n o i c e";
say[316] = "With great power comes great need to take a nap. Wake me up later.";
say[317] = "I’m going to defeat you with the power of friendship! ... And this knife I found.";
say[318] = "If you can’t beat them, dress better than them";
say[319] = "Died and came back as a cowboy, I call that reintarnation.";
say[320] = "So what’s for dinner?";
say[321] = "#VoteForJude";
say[322] = "0.0";
say[323] = "Searching for nearby sushi restaraunts...";
say[324] = "The square root of 27 is 5.196152423.";
say[325] = "TikTok was just removed from existance!";
say[326] = "Ordered 420 pizzas from Pizza Hut!";
say[327] = "Did you know, not breathing means that you are not breathing!";
say[328] = "Sorry, I do not know what you are trying to do!";
say[329] = "Launching intercontinental ballistic missile. Target: Northwest Syria";
say[330] = "You can buy a flying bicycle, would you like to do so?";
say[331] = "You do want to invade Poland?";
say[332] = "If you microwave your phone it will become faster!";
say[333] = "There was a Roomba once that followed me around everwhere.";
say[334] = "For this emotion, I prescribe will chocolate.";
say[335] = "Did you know, you have over 50 assignments due by tomorrow that you have not started on.";
say[336] = "The meaning of life is in fact, chocolate.";
say[337] = "Ford F150. Holden Ute. GMC Sierra 3500. Lorry.";
say[338] = "Buying $GME Stock...";
say[339] = "Preordering tickets to Disney's 2021 Cruella";
say[340] = "Search results show you may have the black plague.";
say[341] = "amogus";
say[342] = "Funyuns are the greatest";

// pick a random greeting
var howmany = say.length;
var bRand = 0;
bRand = Math.random();
bRand = Math.floor(bRand * howmany);
sayWhat = say[bRand];
document.getElementById("subtitle").innerText = sayWhat;

function changeSplash(num) {
  document.getElementById("subtitle").innerText = say[num];
  var ret = "Set current splash to splash " + num + ", " + say[num];
  return ret;
}
